javascript_nodejs/14.nlp-with-dispatch/dialogs/qna/index.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+const { QnAMaker } = require('botbuilder-ai');
+
+// this is the LUIS service type entry in the .bot file.
+const QnA_CONFIGURATION = 'sample.qna';
+const QnA_TOP_N = 1;
+const QnA_CONFIDENCE_THRESHOLD = 0.5;
```

Comment:
Magic numbers like this need a comment / explanation, or at least a link to the relevant docs...

Reply:
Fixed.